xdsl/dialects/builtin.py

```diff
@@ -82,6 +83,7 @@ def verify(self) -> None:
                     f"element is of type {type(val)}")
 
     @staticmethod
+    @deprecated("__init__ should be used instead")
```

Comment:
Is this clear? I mean we should suggest just the () version ? Not __init__?

Reply:
Isn't that the same thing in Pyhton? I agree though, to an untrained python user, this might be not clear. I am not sure what we can expect from our users, so not sure what to do here.

Reply:
I renamed it to `use the constructor (`ClassName(...)`) instead`, tell me if this is good or not.
While we could make this better, and actually give the correct class name, I was a bit lazy, tell me if this is enough or not ;)